pkg/doc/verifiable/test_suite/verifiable_suite_test.go

```diff
@@ -0,0 +1,162 @@
+/*
```

Comment:
Underscore in package name is a bit unusual.

Reply:
Will be fixed!

Reply:
Fixed.